CI failed at generating CA pair

```
=== RUN   TestEnd2EndMySQL
--- FAIL: TestEnd2EndMySQL (0.17s)
    main_test.go:241: failed to generate CA pair exit status 1
=== RUN   TestEnd2EndHive
--- FAIL: TestEnd2EndHive (0.06s)
    main_test.go:273: failed to generate CA pair exit status 1
=== RUN   TestEnd2EndMaxCompute
--- FAIL: TestEnd2EndMaxCompute (0.39s)
    main_test.go:300: failed to generate CA pair exit status 1
=== RUN   TestEnd2EndMaxComputeALPS
--- FAIL: TestEnd2EndMaxComputeALPS (0.22s)
    main_test.go:334: failed to generate CA pair exit status 1
=== RUN   TestEnd2EndMaxComputeElasticDL
--- FAIL: TestEnd2EndMaxComputeElasticDL (0.20s)
    main_test.go:373: failed to generate CA pair exit status 1
FAIL
coverage: 0.0% of statements
FAIL	github.com/sql-machine-learning/sqlflow/cmd/sqlflowserver	1.054s
```

--- a/cmd/sqlflowserver/main_test.go
+++ b/cmd/sqlflowserver/main_test.go
@@ -199,15 +199,15 @@ func generateTempCA() (tmpDir, caCrt, caKey string, err error) {
 	caCrt = path.Join(tmpDir, "ca.crt")
 	if output, err := exec.Command("openssl", "genrsa", "-out", caKey, "2048").CombinedOutput(); err != nil {
 		err = fmt.Errorf("\n%s\n%s", output, err.Error())
-		return
+		return "", "", "", err
 	}
 	if output, err := exec.Command("openssl", "req", "-nodes", "-new", "-key", caKey, "-subj", "/CN=localhost", "-out", caCsr).CombinedOutput(); err != nil {
 		err = fmt.Errorf("\n%s\n%s", output, err.Error())
-		return
+		return "", "", "", err
 	}
 	if output, err := exec.Command("openssl", "x509", "-req", "-sha256", "-days", "365", "-in", caCsr, "-signkey", caKey, "-out", caCrt).CombinedOutput(); err != nil {
 		err = fmt.Errorf("\n%s\n%s", output, err.Error())
-		return
+		return "", "", "", err
 	}
 	os.Setenv("SQLFLOW_CA_CRT", caCrt)
 	os.Setenv("SQLFLOW_CA_KEY", caKey)

--- a/cmd/sqlflowserver/main_test.go
+++ b/cmd/sqlflowserver/main_test.go
@@ -197,13 +197,16 @@ func generateTempCA() (tmpDir, caCrt, caKey string, err error) {
 	caKey = path.Join(tmpDir, "ca.key")
 	caCsr := path.Join(tmpDir, "ca.csr")
 	caCrt = path.Join(tmpDir, "ca.crt")
-	if err = exec.Command("openssl", "genrsa", "-out", caKey, "2048").Run(); err != nil {
+	if output, err := exec.Command("openssl", "genrsa", "-out", caKey, "2048").CombinedOutput(); err != nil {
+		err = fmt.Errorf("\n%s\n%s", output, err.Error())
 		return
 	}
-	if err = exec.Command("openssl", "req", "-nodes", "-new", "-key", caKey, "-subj", "/CN=localhost", "-out", caCsr).Run(); err != nil {
+	if output, err := exec.Command("openssl", "req", "-nodes", "-new", "-key", caKey, "-subj", "/CN=localhost", "-out", caCsr).CombinedOutput(); err != nil {
+		err = fmt.Errorf("\n%s\n%s", output, err.Error())
 		return
 	}
-	if err = exec.Command("openssl", "x509", "-req", "-sha256", "-days", "365", "-in", caCsr, "-signkey", caKey, "-out", caCrt).Run(); err != nil {
+	if output, err := exec.Command("openssl", "x509", "-req", "-sha256", "-days", "365", "-in", caCsr, "-signkey", caKey, "-out", caCrt).CombinedOutput(); err != nil {
+		err = fmt.Errorf("\n%s\n%s", output, err.Error())
 		return
 	}
 	os.Setenv("SQLFLOW_CA_CRT", caCrt)

